dev/workflow/rocoto/gefs_tasks.py

```diff
@@ -61,13 +108,16 @@ def prep_emissions(self):
 
     def fcst(self):
         dependencies = []
-        dep_dict = {'type': 'task', 'name': f'{self.run}_stage_ic'}
-        dependencies.append(rocoto.add_dependency(dep_dict))
 
+        if self.app_config.gefstype in ['gefs-offline']:
+            dep_dict = {'type': 'task', 'name': f'{self.run}_stage_ic'}
+            dependencies.append(rocoto.add_dependency(dep_dict))
+        if self.app_config.gefstype in ['near-real-time']:
+            dep_dict = {'type': 'task', 'name': f'{self.run}_stage_ic_mem000'}
+            dependencies.append(rocoto.add_dependency(dep_dict))
```

Comment:
Minor nitpick.  This should probably be an `elif`:
```suggestion
        elif self.app_config.gefstype in ['near-real-time']:
            dep_dict = {'type': 'task', 'name': f'{self.run}_stage_ic_mem000'}
            dependencies.append(rocoto.add_dependency(dep_dict))
```